### ISSUE TITLE
Indent with tab issue

### ISSUE THREAD

**Issue by clarkWGriswold** [beautify-web/js-beautify#1470]
Beautifying with "Indent with a tab character" is indenting with a single space, see below.
```
answer = checkForHardFreezeBlackoutSchedules();

function checkForHardFreezeBlackoutSchedules() {

 var blackoutHelper = new TUChangeBlackoutHelper();
 var startDate = new GlideDateTime(current.start_date);
 var endDate = new GlideDateTime(current.end_date);

 var blckOutSchedGR = new GlideRecord('cmn_schedule_blackout');
 blckOutSchedGR.addQuery('u_type_of_freeze', 'Hard');
 blckOutSchedGR.query();

 while (blckOutSchedGR.next()) {

  if (blackoutHelper.isTimeFrameInBlackoutWindows(startDate, endDate, String(blckOutSchedGR.sys_id), true)) {

   return 'yes';
  }
 }
 return 'yes';
 //return 'no';
}
```

**Comment by bitwiseman** [comment]
@clarkWGriswold  where are you seeing this issue (website, nodejs, python, some editor plugin)?  What version of beautifier are you using?

**Comment by templarundead** [comment]
Website v1.8.0-rc4

**Comment by clarkWGriswold** [comment]
Sorry, it was in the website.

**Comment by bitwiseman** [comment]
@clarkWGriswold 
There's problems displaying tabs on webpages. 
I'm not sure how to fix this yet.  It maybe a tab display issue or it may be that I have an old version of codemirror. 
https://stackoverflow.com/questions/3946688/encoding-a-tab-in-html

**Comment by clarkWGriswold** [comment]
I think the value the "Indent with a tab character" is setting is
incorrect. When I select this option, the "Your Selected Options (JSON)"
shows the indent_char is " " (single space). When I change this to "\t" it
works correctly.

I hope that helps. I use this site a lot though-out the day.

**Comment by bitwiseman** [comment]
Hmm ...
https://github.com/beautify-web/js-beautify/blob/master/index.html#L78 just sets the value to 1. 
https://github.com/beautify-web/js-beautify/blob/master/web/common-function.js#L153 sets the indent char to `\t` but I'm wondering if I should make it `\\t`.

**Comment by bitwiseman** [comment]
Oh, I see it: 
https://github.com/beautify-web/js-beautify/blob/master/web/common-function.js#L152 gets the value of the web pages setting, which is a string.  Then 153, checks if that is equal to a number. 😱

**Comment by bitwiseman** [comment]
@clarkWGriswold 
Thanks for your help.  I've cherry-picked this fix to the website, so it should be working now.

**Comment by clarkWGriswold** [comment]
great. it looks like it's working. I was poking around a little because I
use JsFormat in sublime text, I found there's another option named
"indent_with_tabs". When this is set to true, it overrides the indent_size
and indent_car options. I set the "Additional Settings (JSON)" field on the
website to:
{
"indent_with_tabs": true
}

then set to Index with 2 spaces. When I beautify with these settings, it
indents with tab. If I change the setting to false, it indents with 2
spaces. Since I configured my JsFormat to use my settings from the
jsbeautifier site, I think maybe this option just fell off the options
setting. Obviously, I don't know what's going on in the background, just a
though I had.

I appreciate your site. Like I said, i use it a lot throughout the day and
I recommend it to everyone that will listen to me.

Thank you!
